rosa_hcp/rosa-hcp-cluster-no-cni.adoc

```diff
@@ -10,11 +10,20 @@ toc::[]
 You can use your own Container Network Interface (CNI) plugin when creating a {hcp-title-first} cluster.
 You can create a {hcp-title} cluster without a CNI and install your own CNI plugin after cluster creation.
 
-[NOTE]
+[IMPORTANT]
 ====
 For customers who choose to use their own CNI, the responsibility of CNI plugin support belongs to the customer in coordination with their chosen CNI vendor.
 ====
 
+The default plugin for {hcp-title} is the xref:../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes network plugin]. This plugin is the only supported CNI plugin for {hcp-title}.  
+
+If you choose to use your own CNI for {hcp-title} clusters, it is strongly recommended that you obtain commercial support for the plugin before creating your clusters. Red Hat support cannot assist with CNI-related issues such as pod to pod traffic for customers who choose to use their own CNI. Red Hat still provides support for all non-CNI issues. If you want CNI-related support from Red Hat, you must use the OVN-Kubernetes network plugin. See the xref:../rosa_architecture/rosa_policy_service_definition/rosa-policy-responsibility-matrix.adoc#rosa-policy-responsibility-matrix[responsibility matrix] for more information.
```

Comment:
@bchandra-ocp - Are there other examples of CNI-related issues besides pod to pod traffic? Thanks!

Reply:
I'd defer to @wgordon17 for additional examples. It's going to depend on our ability to call what's CNI related issue and what's not.

Reply:
None that I'm aware of